dashboard/README.md

```diff
@@ -0,0 +1,65 @@
+# This is a simple guide for building an operator with kubebuilder version 2
+
+## 1. Create a folder
+```bash
+mkdir dashboard
+cd dashboard
+```
+
+## 2. Enable kubebuilder plugins and init
+```bash
+export KUBEBUILDER_ENABLE_PLUGINS=1
+kubebuilder init --fetch-deps=false --domain=x-k8s.io --license=apache2
+kubebuilder create api --pattern=addon --controller=true --example=false --group=addons --kind=Dashboard --make=false
+ --namespaced=true --resource=true --version=v1alpha1
+```
+
+## 3. Run go mod vendor:
+
+```bash
+go mod vendor
+```
+
+## 3. Delete unnecessary test files
+Delete the test suites that are more checking that kubebuilder is working:
+
+```bash
+find . -name "*_test.go" -delete
+```
+
+# Determine the a stable version and include its manifest:
+
+Make a directory under `channels/packages/dashboard` with the version number. For the dashboard operator the stable
+ version was `2.0.0`
+```bash
+cd  channels/packages/dashboard
+mkdoir 2.0.0
+```
+
+Download the manifest yaml
+```bash
+cd 2.0.0
+wget kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0/aio/deploy/recommended.yaml
+mv recommended.yaml manifest.yaml
+```
```

Comment:
Caught some small typos here /w `mkdoir` and `kubectl apply`.

It can be a bit clunky to ask a user to `cd` into directories, because you need to instruct them to `cd` back out.
Here's a simpler method:
`````suggestion
Make a directory under `channels/packages/dashboard` for a stable version of the dashboard -- then download the manifest for it:
```bash
mkdir -p channels/packages/dashboard/2.0.0
wget -O channels/packages/dashboard/2.0.0/manifest.yaml https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0/aio/deploy/recommended.yaml
```

`````